packages/syntax-highlighter/__tests__/__fixtures__/objc/sample.m

```diff
@@ -0,0 +1,14 @@
+/*
+ Build on OS X:
+ clang -framework Foundation -fobjc-arc objc.m -o objc
+
+ Build on Linux with GNUstep:
+ clang \`gnustep-config --objc-flags\` \`gnustep-config --base-libs\` -fobjc-nonfragile-abi -fobjc-arc objc.m -o objc
+ */
+
+#import <Foundation/Foundation.h>
+
+int main(void)
```

Comment:
Do you need to pass in a `void` arg here? I think if you're trying to say it has no return value, the syntax is a bit different.

Reply:
Pulled this example from https://github.com/leachim6/hello-world/blob/master/o/objective-c.m, which reminds me I need to pull in their license.